pkg/apis/v1alpha1/deliverable.go

```diff
@@ -44,6 +44,10 @@ const (
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Source",type="string",JSONPath=`.spec.source['git.url','image']`
+// +kubebuilder:printcolumn:name="Delivery",type="string",JSONPath=".status.deliveryRef.name"
+// +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=`.status.conditions[?(@.type=='Ready')].status`
+// +kubebuilder:printcolumn:name="Reason",type="string",JSONPath=`.status.conditions[?(@.type=='Ready')].reason`
```

Comment:
should we add age as well?

```
// +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
```

Reply:
If you wanted a time value, I'd think most recent status change would be more interesting.

Reply:
I'm not going to make this change right now, but feel free to do this in a subsequent PR; I also didn't do anything for any of the `Cluster*` resources.